enforceBeachball.js

```diff
@@ -0,0 +1,12 @@
+const cp = require('child_process');
+
+cp.exec('yarn changefile < killProcessCommand.txt', (err, stdout, stderr) => {
```

Comment:
What is the killProcessCommand.txt file? Can this run without it?

Reply:
It just contains ^C. When you run yarn beachball, it gives one of two responses: it either says you don't need a changefile and terminates or it prompts you for an answer. The ^C in the killProcessCommand.txt file makes it so it doesn't hang there forever waiting for an answer and time out the check. I experimented with timeouts and having a timeout of 3~5 seconds also works, but I thought this made more sense. What do you think is better? (Or maybe you have a better option?)

Reply:
We discussed offline and thought the timeout was better :) switching to timeout